### ISSUE TITLE
Images on docs should be compressed

### ISSUE THREAD

**Issue by ScottRobbins** [qutheory/red-feedback#21]
Some of the images, at least on the Quick Start page could use some compression. That page took a couple seconds to load for me with my usually decent internet.

**Comment by joscdk** [comment]
Yeah i have seen this can be very slow loading :) .. I will look into compressing them :)
